src/sanitizer.js

```diff
@@ -308,7 +326,17 @@ export function sanitizeHtml(html) {
  */
 export function sanitizeFormattingHtml(html) {
   return htmlSanitizer.sanitizeWithPolicy(html,
-      function(tagName, unusedAttrs) {
+      function(tagName, attribs) {
+        if (tagName == 'template') {
```

Comment:
Often `element.tagName` is all uppercase. Where does this tagName value come from, and is it guaranteed to always be lowercase?

Reply:
tagName here is passed from Caja, so guaranteed to be lowercase